cloud-operations/README.md

```diff
@@ -50,5 +50,8 @@ The example's feed tracks changes to Google Compute instances, and the Cloud Fun
 
 
 This [example](./onprem-sa-key-management) shows how to manage IAM Service Account Keys by manually generating a key pair and uploading the public part of the key to GCP.
-s
+<br clear="left">
+
+## Migrate for Compute Engine (v5)
+<a href="./vm-migration/host-target-projects/diagram.png" title="Packer image builder"><img src="./packer-image-builder/diagram.png" align="left" width="280px"></a> This set of [examples](./vm-migration) show how to deploy Migrate for Compute Engine (v5) on top of existing Cloud Foundations on different scenarios. An example on how to deploy M4CE (v5) migrate connetor on ESXi is also part of the example.
```

Comment:
Nit: grammar
`[...]show_s_ how to deploy[...]`
`An example on how to deploy M4CE (v5) migrate conne_c_tor on ESXi is also _included_`